src/ml-search.service.js

```diff
@@ -629,9 +629,9 @@
      */
     getCombinedQuery: function getCombinedQuery(includeOptions) {
       var d = $q.defer(),
-          combined = {
-            search: { query: this.getQuery() }
-          };
+        combined = {
+          search: { query: this.getQuery() }
+        };
 
```

Comment:
I'm sure the TextMate plugin looks at .jshintrc in the project folder, but which version it is using, is a bit hard to track. From github it says it is using node, so I would expect to leverage the same version as I see with $ jshint --version. That returns v2.8.0..